st2common/st2common/rbac/types.py

```diff
@@ -38,8 +38,8 @@ class PermissionType(Enum):
     PACK_ALL = 'pack_all'
 
     # Note: Right now we only have read endpoints for sensors types
-    SENSOR_VIEW = 'sensor_view'
-    SENSOR_ALL = 'sensor_all'
+    SENSOR_VIEW = 'sensor_type_view'
```

Comment:
I'm fine with this change for now, I can figure something else out in the future PRs if needed.